src/Views/Game.hs

```diff
@@ -39,6 +39,8 @@ createGameDOM api gapi = do
     [
       prop "id" =: "startGameButton"
     ]
+  setClass createStartGameBtn "btn" True
```

Comment:
This can be done directly in the attributes on the lines above.
```
createStartGameBtn <- newElem "button" `with`
    [
        attr "id" =: "start-game-button,
        attr "class" =: "btn btn-default"
    ]
```
This goes for just about everything you've done in this commit, sorry :P

Also note that I used `attr` in the above example and `prop` is currently being used for the `createStartGameBtn`. We should be consistent... It's an attribute when written in HTML but a property when in the DOM objects. I don't know what we should call them here... but I think `attr` is more commonly used throughout our project. Or maybe i'm just biast.